config/config.go

```diff
@@ -694,13 +694,13 @@ type P2PConfig struct { //nolint: maligned
 	// Force dial to fail
 	TestDialFail bool `mapstructure:"test-dial-fail"`
 
-	// DisableLegacy is used mostly for testing to enable or disable the legacy
+	// UsesLegacy is used mostly for testing to enable or disable the legacy
```

Comment:
```suggestion
	// UseLegacy is used mostly for testing to enable or disable the legacy
```

Reply:
Could we also please make this say what it does, rather than merely what it's for? If someone needs to enable this, we could also use it as a way to get feedback. Perhaps:

```go
// UseLegacy enables the "legacy" P2P implementation deprecated in release 0.34.x
// and disables the newer default implementation. This flag will be removed in a future
// release. If you need to enable this, please consider filing an issue to tell us why at
// https://github.com/tendermint/tendermint/issues
``` 

or words to that effect.